deployment pipeline: cannot `jb install kube-prometheus`

Seen, for example, [here](https://github.com/conbench/conbench/actions/runs/6318525930/job/17157695989).

When running this command:

https://github.com/conbench/conbench/blob/89ddc4b5745c8bbef74900a5a8ff7d4f51caa0d9/Makefile#L316-L317

we see

```
GET https://github.com/pyrra-dev/pyrra/archive/f2052cce0c848bb8c4c4d5d964a97b5038237487.tar.gz 200
panic: rename vendor/.tmp/jsonnetpkg-github.com-pyrra-dev-pyrra-config-crd-bases-main336769959/config/crd/bases vendor/github.com/pyrra-dev/pyrra/config/crd/bases: no such file or directory
```

I think there might be something wrong with the vendored `pyrra` repo? We got a different tar the [last time](https://github.com/conbench/conbench/actions/runs/6303291179/job/17112233182) the run succeeded.

--- a/Makefile
+++ b/Makefile
@@ -311,10 +311,10 @@ set-build-info:
 .PHONY: jsonnet-kube-prom-manifests
 jsonnet-kube-prom-manifests:
 	mkdir -p _kpbuild && cd _kpbuild  && mkdir -p cb-kube-prometheus
-	cd _kpbuild/cb-kube-prometheus && echo '{"version": 1, "dependencies": [], "legacyImports": true}' > jsonnetfile.json
+	cd _kpbuild/cb-kube-prometheus && git clone https://github.com/prometheus-operator/kube-prometheus . &&	git checkout 7fafc4cadc1
 	cd _kpbuild/cb-kube-prometheus && \
 		time docker run --rm -v $$(pwd):$$(pwd) --workdir $$(pwd) quay.io/coreos/jsonnet-ci \
-			sh -c 'jb install github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus@d3889807798d && chmod -R 777 *'
+			sh -c 'jb install && chmod -R 777 *'
 	cd _kpbuild/cb-kube-prometheus && /bin/ls -ahl
 	cp k8s/kube-prometheus/conbench-flavor.jsonnet _kpbuild/cb-kube-prometheus
 	cp k8s/kube-prometheus/conbench-grafana-dashboard.json _kpbuild/cb-kube-prometheus
@@ -353,7 +353,7 @@ jsonnet-kube-prom-manifests:
 		fi
 	cat _kpbuild/cb-kube-prometheus/conbench-flavor.jsonnet
 	cd _kpbuild/cb-kube-prometheus && \
-		wget https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/d3889807798d/build.sh -O build.sh
+		wget https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/7fafc4cadc1/build.sh -O build.sh
 	cd _kpbuild/cb-kube-prometheus && \
 		time docker run  --rm -v $$(pwd):$$(pwd) --workdir $$(pwd) quay.io/coreos/jsonnet-ci \
 			sh -c 'bash build.sh conbench-flavor.jsonnet && chmod -R 777 *'

